server/docker-compose.yml

```diff
@@ -1,43 +1,14 @@
 services:
   web:
     image: IMAGE_REPLACE_ME
-    depends_on:
-      - refinery
     environment:
       - BEANSTALK_PORT=80
       - PRODUCTION=true
-      - HONEYCOMB_ENDPOINT=http://localhost:8082
+      # Internal load balancer
+      - HONEYCOMB_ENDPOINT=http://refinery.us-east-1.elasticbeanstalk.com:8082
```

Comment:
oh cool! Curious q: how does the domain work? i.e did we nab refinery.us-east-1 ? Or can other people have this endpoint within their networks?

Reply:
It's all ours--the elastic beanstalk cnames are all in one global namespace (well, a namespace per region) and you keep it as long as you don't terminate the environment.

Reply:
Heeck yeah!!